products/jira.md

```diff
@@ -143,6 +143,54 @@ releases:
     link:
       https://confluence.atlassian.com/jirasoftware/jira-software-8-20-x-release-notes-1086411771.html
 
+-   releaseCycle: "8.19"
+    releaseDate: 2021-08-25
+    eol: 2023-08-26
+    latest: "8.19.1"
+    latestReleaseDate: 2021-09-14
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-19-1-1086395933.html
+
+-   releaseCycle: "8.18"
+    releaseDate: 2021-07-01
+    eol: 2023-07-01
+    latest: "8.18.2"
+    latestReleaseDate: 2021-08-17
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-18-2-1077915228.html
+
+-   releaseCycle: "8.17"
+    releaseDate: 2021-05-17
+    eol: 2023-05-18
+    latest: "8.17.1"
+    latestReleaseDate: 2021-06-14
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-17-1-1069027535.html
+
+-   releaseCycle: "8.16"
+    releaseDate: 2021-03-22
+    eol: 2023-03-23
+    latest: "8.16.2"
+    latestReleaseDate: 2021-06-01
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-16-2-1063177762.html
+
+-   releaseCycle: "8.15"
+    releaseDate: 2021-01-21
+    eol: 2023-02-02
+    latest: "8.15.1"
+    latestReleaseDate: 2021-03-21
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-15-1-1044113285.html
+
+-   releaseCycle: "8.14"
+    releaseDate: 2020-11-22
+    eol: 2022-11-23
+    latest: "8.14.1"
+    latestReleaseDate: 2021-01-13
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-14-1-1041828920.html
+
 -   releaseCycle: "8.13"
     releaseDate: 2020-10-06
     eol: 2022-11-08
```

Comment:
```suggestion
    eol: 2022-10-08
```

See https://web.archive.org/web/20211117125223/https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html.